browser_patches/webkit/build.sh

```diff
@@ -11,15 +11,15 @@ if [[ "$(uname)" == "Darwin" ]]; then
 elif [[ "$(uname)" == "Linux" ]]; then
   cd "checkout"
   if [[ "$1" == "--wpe" ]]; then
-    if ! [[ -d ./WebKitBuild/DependenciesWPE ]]; then
-      yes | DEBIAN_FRONTEND=noninteractive ./Tools/Scripts/update-webkitwpe-libs
+    if ! [[ -d ./WebKitBuildWPE/DependenciesWPE ]]; then
+      yes | WEBKIT_OUTPUTDIR=./WebKitBuildWPE DEBIAN_FRONTEND=noninteractive ./Tools/Scripts/update-webkitwpe-libs
```

Comment:
When I tried it, update-webkitwpe-libs did not support WEBKIT_OUTPUTDIR. Having said that, we could try using WebKitBuild/ for both deps and only split the release folders.